db/migrations/01_create_chitters_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE chitters(id SERIAL PRIMARY KEY, url VARCHAR(60));
```

Comment:
This table is not referenced in the rest of the code, a table called 'peeps' with a column 'message' is though.